client.go

```diff
@@ -163,20 +159,6 @@ func (c Client) AuthURL() string {
 	return url
 }
 
-// TokenURL returns the token url for the environment
-func (c Client) TokenURL() string {
```

Comment:
This is an API change and would require a major version bump. I'd like to keep this method and the constants intact for anyone relying on them in this version.